### PR TITLE
User can input preferences for the genres of anime he or she enjoys watching.

### PR DESCRIPTION
**Pull request by ALee-DF**
user can input preferences

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -47,4 +52,53 @@ app.post('/command', (req, res) => {
...
+    method: 'POST',
+    headers: {
+      'Content-type': 'application/json'
+    },
+    json: JSONmessage
+  }
+  request(postOptions, (err, res, body) => {
+    if (err) {
+      console.error(err)
+      res.sendStatus(500)
```
Comment (thebearingedge) [inline index.js:67]: This `res` object is the response _received_ by `request`, not a `res` that `express` uses to _send_ responses.

File: index.js [hunk 2]
```diff
@@ -47,4 +52,53 @@ app.post('/command', (req, res) => {
...
+  })
+}
+
+app.post('/checklist', urlencodedParser, (req, res) => {
+  res.status(200).end()
+  const reqBody = req.body
+  const responseURL = reqBody.response_url
+  if (reqBody.token !== animebotVerificationToken) {
+    console.error('Access Forbidden')
+    res.sendStatus(403)
```
Comment (thebearingedge) [inline index.js:78]: You can't respond here anymore, you already did on line `73`.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Great job so far! Where is the part where you actually save the preferences?

**Comment by ALee-DF** [comment]
For the part I save the preferences, that's gonna be under #11 where the user can update (and subsequently save) his or her preferences into the MongoDB database.

**Pull request closed without merging**